src/App.tsx

```diff
@@ -5,6 +5,7 @@ import { Route, Routes } from "react-router-dom";
 import "../src/_styles/index.scss";
 import AboutMe from "./pages/AboutMe";
 import HomePage from "./pages/HomePage";
+import NoMachPage from "./pages/NoMachPage";
```

Comment:
Let's add alias for /pages rout "@pages" like it implemented for @components. https://github.com/AnastasiSo/Portfolio-Project/blob/884e034af722f265d04e3a0d9e32de916c762f5d/webpack.config.js#L40

Reply:
- [ ] Done [aab8067](https://github.com/AnastasiSo/Portfolio-Project/pull/27/commits/aab806757f8426c4827d1201b10ad8aac5e12d23) and [2a41875](https://github.com/AnastasiSo/Portfolio-Project/pull/27/commits/2a41875f2ea89005e1748cfd4a6182bbda3f6af6).